pkg/local_object_storage/blobstor/compression/compress.go

```diff
@@ -0,0 +1,102 @@
+package compression
+
+import (
+	"bytes"
+	"strings"
+
+	"github.com/klauspost/compress/zstd"
+	objectSDK "github.com/nspcc-dev/neofs-sdk-go/object"
+)
+
+// CConfig represents common compression-related configuration.
+type CConfig struct {
```

Comment:
Why double `C`?

Reply:
It was meant as `Compression Config` and embedded struct named `Config` would be rather misleading. I guess I'll just remove confusing embeddings and use `compression.Config` as a name.